themes-default/slim/views/addShows_newShow.mako

```diff
@@ -370,9 +370,8 @@ const startVue = () => {
                         alreadyAdded = (() => {
                             if (!alreadyAdded) return false;
                             // Extract existing show info
-                            const [ mIndexerId, mShowId ] = alreadyAdded;
-                            const indexerIdentifier = indexers[mIndexerId] ? indexers[mIndexerId].identifier : mIndexerId;
-                            return 'home/displayShow?indexername=' + indexerIdentifier + '&seriesid=' + mShowId;
+                            const [ mIndexerName, mShowId ] = alreadyAdded;
+                            return 'home/displayShow?indexername=' + mIndexerName + '&seriesid=' + mShowId;
```

Comment:
Just updating the code here to account for the changes in backend.
No need to get the indexer name, as we already know it.